src/main/java/org/maxgamer/rs/core/server/IllegalThreadException.java

```diff
@@ -4,13 +4,13 @@
  * @author netherfoam
  */
 public class IllegalThreadException extends RuntimeException {
-	private static final long serialVersionUID = -7372301502938861928L;
-	
-	public IllegalThreadException(String msg) {
-		super(msg);
-	}
-	
-	public IllegalThreadException() {
-		super();
-	}
+    private static final long serialVersionUID = -7372301502938861928L;
+
+    public IllegalThreadException(String msg) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://titan.maxgamer.org:9000/coding_rules#rule_key=squid%3AUndocumentedApi)